apps/cookbook/src/app/examples/segmented-control-example/default/default.ts

```diff
@@ -115,7 +115,15 @@ export class SegmentedControlExampleDefaultComponent implements OnInit {
         themeColor: 'warning',
       },
     },
-    { text: 'Second item', id: 'second' },
+    {
+      text: 'Second item',
```

Comment:
I think we should display the `defaultItems` as part of the example in the documentation. Especially now that the `badge` field has gotten a tad more complex.

Reply:
Good idea, added!